models/index.js

```diff
@@ -0,0 +1,129 @@
+var pass = '1234';
+
+var Sequelize = require('Sequelize');
+var orm = new Sequelize('ChallengeDb', 'app', pass, {
+  host: 'localhost',
+  dialect: 'sqlite',
+
+  pool: {
+    max: 5,
+    min: 0,
+    idle: 3000
+  },
+
+  storage: '../db/ChallengeDb.sqlite'
+});
+
+orm.authenticate()
+  // .then(function() {
+  //   console.log('Connection to db successful!');
+  //  })
+  .catch(function(err) {
+    console.log('Connection to db failed: ', err);
+  })
+  .done();
+
+var Users = orm.define('user', {
+  user_id: {
+    type: Sequelize.INTEGER,
+    autoincrement: true,
+    primaryKey: true
+  },
+
+  first_name: {
```

Comment:
There's no need to make objects with just one or two properties span multiple lines.